openreview/venue/matching.py

```diff
@@ -1063,17 +1063,7 @@ def setup(self, compute_affinity_scores=False, compute_conflicts=False, compute_
 
             invitation = openreview.tools.get_invitation(self.client, venue.get_bid_id(self.match_group.id))
             if invitation:
-                score_spec[invitation.id] = {
-                    'weight': 1,
-                    'default': 0,
-                    'translate_map' : {
-                        'Very High': 1.0,
-                        'High': 0.5,
-                        'Neutral': 0.0,
-                        'Low': -0.5,
-                        'Very Low': -1.0
-                    }
-                }
+                score_spec[invitation.id] = invitation.content['scores_spec']['value']
```

Comment:
@melisabok what do you think about this? I wonder the same thing, but it may not be a big issue if bidding isn't commonly used

Reply:
We can leave this as it its, I didn't want to hav the translate_map duplicated in the code, that is why I suggested to store it in the invitation.content.

Sometimes a venue may want to use a different scale or bid labels so it would be nice this can be configure by themselves.